src/platform-includes/getting-started-config/javascript.remix.mdx

```diff
@@ -144,6 +144,46 @@ withSentry(App, {
 });
 ```
 
+## Remix v2 features
+
+_Available from version [...]_
+
+[Remix v2](https://remix.run/docs/en/main/pages/v2) will introduce new features that require additional configuration to work with Sentry. These features are also available from version [1.17.0](https://github.com/remix-run/remix/releases/tag/remix%401.17.0) with [future flags](https://remix.run/docs/en/main/pages/api-development-strategy#current-future-flags). Sentry automatically detects if you are using `v2_errorBoundary` flag and changes its behaviour accordingly.
```

Comment:
l: Given that we're talking about the error boundary in the subsection below, I'd remove the sentence here.
In general, I'm wondering if this (the detection of the v2 error boundary) is something users need to know about? Or if it causes confusion and they think that they don't need to explicitly call `captureRemixErrorBoundary` as shown below.

```suggestion
[Remix v2](https://remix.run/docs/en/main/pages/v2) will introduce new features that require additional configuration to work with Sentry. These features are also available from version [1.17.0](https://github.com/remix-run/remix/releases/tag/remix%401.17.0) with [future flags](https://remix.run/docs/en/main/pages/api-development-strategy#current-future-flags). 
```

Reply:
Good point, yes that's too much detail, and might create confusion.